README.md

```diff
@@ -54,6 +54,13 @@ Thanks goes to these wonderful people ([:hugs:](https://allcontributors.org/docs
                     <sub><b>Ali Solanki</b></sub>
                 </a> 
             </td>
+            <td align="center">
+                <a href="https://github.com/BhavyashKoluguri">
+                    <img src="https://avatars.githubusercontent.com/u/102355627?v=4" width="100px;" alt="Ali Solanki"/>
```

Comment:
```suggestion
                    <img src="https://avatars.githubusercontent.com/u/102355627?v=4" width="100px;" alt="Koluguri Bhavyash"/>
```

Reply:
The name should be added at the end of the table